pyproject.toml

```diff
@@ -0,0 +1,42 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+packages = ["archive_verify"]
+
+[project]
+name = "archive-verify"
+version = "1.1.0"
+authors = [
+    {name = "SNP&SEQ Technology Platform, Uppsala University", email = "seq@medsci.uu.se" },
+]
+keywords = ["bioinformatics"]
+description = "Micro-service for downloading and verifying archives with DSMC"
+readme = "README.md"
+license = {file = "LICENSE"}
+requires-python = ">=3.7"
```

Comment:
This is inconsistent with what it says in the README (there it says 3.9).

Reply:
fair enough, at some point I felt a bit conflicted about what to base the requirement on (other than what my current dev environment uses) and I think 3.7 was what I found among dependencies... anyways, thanks for pointing out, I'll fix the inconsistency